src/compiler/crystal/tools/formatter.cr

```diff
@@ -4585,8 +4585,14 @@ module Crystal
               @doc_comments << current_doc_comment if current_doc_comment.needs_format
               @current_doc_comment = nil
             else
+              # Normalize crystal language tag
+              if chopped = (value.rchop?("cr") || value.rchop?("crystal"))
+                value = chopped
+                language = ""
+              end
```

Comment:
Wouldn't this normalize ` ```bettercrystal` to ` ```better` etc? 🤔

Reply:
Oh right. I wanted to avoid dealing with reconstruction of `value` with leading spaces and stuff. Should be fine with an additional confirmation step.